app/models/judicial_result.rb

```diff
@@ -23,9 +23,7 @@ class JudicialResult < ApplicationRecord
   validates :isAvailableForCourtExtract, inclusion: [true, false]
   validates :orderedDate, presence: true
   validates :category, presence: true, inclusion: CATEGORIES
-  validates :resultText, presence: true
-  validates :terminatesOffenceProceedings, inclusion: [true, false]
-  validates :postHearingCustodyStatus, inclusion: POST_HEARING_CUSTODY_STATUSES, allow_blank: true
```

Comment:
Need to add:
durationElement
parentJudicialResultId
parentJudicialResultTypeId
rootJudicialResultId
rootJudicialResultTypeId
postHearingCustodyStatus
canBeSubjectOfVariation
canBeSubjectOfBreach
resultWording
welshResultWording
level

Reply:
To keep things more straightforward I've not added all the non-required fields. If needed they can be added in the future.